<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1079</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startGame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>610</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Level 1!</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1041</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startGame_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>610</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bebas Neue</family>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Level 2!</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>startGame</zorder>
   <zorder>startGame_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
